driver/docker-container/driver.go

```diff
@@ -103,6 +105,13 @@ func (d *Driver) create(ctx context.Context, l progress.SubLogger) error {
 		hc := &container.HostConfig{
 			Privileged: true,
 			UsernsMode: "host",
+			Mounts: []mount.Mount{
+				{
+					Type:   mount.TypeVolume,
+					Source: d.Name,
```

Comment:
Sorry, didn't see this the first time I looked at it. Is this enough to create volume if it does not exist and reuse it if it does? I wonder if we should add a suffix to `d.Name`? Maybe "-state".

Reply:
> Sorry, didn't see this the first time I looked at it. Is this enough to create volume if it does not exist and reuse it if it does?

Yes it's enough, tested it live to be sure.

> I wonder if we should add a suffix to `d.Name`? Maybe "-state".

Was wondering too about that. I'm ok with a suffix.